src/bitcoin.ts

```diff
@@ -86,7 +86,7 @@ router.get('/fee/:blockchainTime', async (ctx, _next) => {
 
 router.post('/transactions', async (ctx, _next) => {
   const writeRequest = JSON.parse(ctx.body);
-  const requestHandler = () => blockchainService.writeTransaction(writeRequest.anchorString, writeRequest.fee);
+  const requestHandler = () => blockchainService.writeTransaction(writeRequest.anchorString, writeRequest.minimumFee);
```

Comment:
There will not be an upper bound, but this guarantees that your transactions will go through, as opposed to before, you can have a fee that will never get you onto the blockchain.  It shouldn't be a big issue because we have something called a spending monitor now. What it does is it monitors how much you want to spend in say the next week, if I set it to 1btc, then when it reaches that limit, it will give you a warning and stop making transactions. I see the concern so let me know what you think.